src/frontend/src/pages/FlowPage/index.tsx

```diff
@@ -323,6 +323,7 @@ export default function FlowPage({ flow }: { flow: FlowType }) {
 						onDrop={onDrop}
 						onNodesDelete={onDelete}
 						selectNodesOnDrag={false}
+						proOptions={{hideAttribution: true}}
```

Comment:
Any specific reason to remove this? If it is just because of dark mode, maybe we could talk to their team to see how to support it.

Reply:
Let's not deactivate this for now.

Reply:
Done! Attribution is now on bottom right and is working with dark mode.